stdlib/messaging/nats/src/main/ballerina/src/nats/Module.md

```diff
@@ -173,4 +148,14 @@ nats:ConnectionConfig config = {
 // Initializes a connection.
 nats:Connection connection = new("tls://localhost:4222", config = config);
 ```
->**Note:** The default thread pool size used in Ballerina is number of processors available * 2. You can configure the thread pool size by using the `BALLERINA_MAX_POOL_SIZE` environment variable.
\ No newline at end of file
+>**Note:** The default thread pool size used in Ballerina is the number of processors available * 2. You can configure the thread pool size by using the `BALLERINA_MAX_POOL_SIZE` environment variable.
+
+For information on the operations, which you can perform with this module, see the below **Functions**. 
+
+For examples on the usage of the connector, see the following.
+* [Basic Publisher and Subscriber Example](https://ballerina.io/v1-2/learn/by-example/nats-basic-client.html).
```

Comment:
The urls would have to be updated as @praneesha mentioned.

Reply:
Fixed in 6b572e1d2d1697f8c5ff4d00e14801f9bec39cff